addons/missileguidance/functions/fnc_seekerType_MCLOS.sqf

```diff
@@ -0,0 +1,37 @@
+#include "script_component.hpp"
+/*
+ * Author: BaerMitUmlaut
+ * MCLOS guidance "seeker".
+ *
+ * Arguments:
+ * 1: Guidance Arg Array <ARRAY>
+ *
+ * Return Value:
+ * Missile Aim PosASL <ARRAY>
+ *
+ * Example:
+ * [[], [...]] call ace_missileguidance_fnc_seekerType_MCLOS;
+ *
+ * Public: No
+ */
+params ["", "_args"];
+_args params ["_firedEH", "", "", "", "_stateParams"];
+_firedEH params ["", "", "", "", "", "", "_projectile"];
+_stateParams params ["", "_seekerState"];
+_seekerState params ["_initialized"];
+
+if (isNil "_initialized") then {
+    // Missile just launched, reset
+    GVAR(mclosMissile) = _projectile;
+    GVAR(mclosInput) = [0, 0, 0, 0];
+    _seekerState pushBack true;
+};
+
+// Another missile was fired after this one, fly straight
+if (GVAR(mclosMissile) != _projectile) exitWith {[0, 1, 0]};
```

Comment:
```suggestion
if (GVAR(mclosMissile) != _projectile) exitWith {[0, 0, 0]};
```

I think this was supposed to be `_steerVector = [0,1,0] and then use vectorModelToWorld.
Can just return special value [0,0,0] and guidance should do roughly the same thing.

Reply:
`[0, 1, 0]` make it veer off course, whilst `[0, 0, 0]` seems to do the trick.